libraries/nomad_job.rb

```diff
@@ -20,16 +21,40 @@ def initialize(url, job_name)
   end
 
   def running?
-    job = http_json("#{@url}/v1/job/#{@job_name}")
+    job = get_job
     job['Status'] == 'running'
   end
 
+  def job_status
```

Comment:
You are already describing a job, so you can keep it to "status" only I suppose. Then, you write:

```ruby
describe nomad_job(...) do
  its('status') { should eq 'running' }
end
```